src/tiles/item-image.ts

```diff
@@ -0,0 +1,206 @@
+import {
+  css,
+  CSSResultGroup,
+  html,
+  nothing,
+  PropertyValues,
+  LitElement,
+} from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import { TileModel } from '../models';
+
+@customElement('item-image')
+export class ItemImage extends LitElement {
+  @property({ type: Object }) model?: TileModel;
+
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Boolean }) isListTile = false;
+
+  @state() private isDeemphasize = false;
+
+  protected updated(changed: PropertyValues): void {
+    if (changed.has('model')) {
+      this.setDeemphsize();
+    }
+  }
+
+  private setDeemphsize() {
+    this.isDeemphasize =
+      this.model?.collections.includes('deemphasize') ?? false;
+  }
+
+  render() {
+    return html`
+      <div class=${ifDefined(this.imageBoxClass)}>
```

Comment:
i added `ifDefined` so in the DOM it will not be: `<div class>`

Reply:
Great, thanks!